app/controllers/orders/index.js

```diff
@@ -23,6 +23,12 @@ export default Ember.Controller.extend(SearchMixin, {
     this.cache = new Cache();
   },
 
+  searchProps: {
```

Comment:
Are we fetching `messages` in `index` page as well ?

Reply:
Yes, we will only fetch messages (not message associations) to display the count of unread messages on each order.

Previously, gave a try to fetch count of unread messages from the API serializer response, but again we need to handle live update of that count on new message and also when messages are marked read. But that will add unnecessary observers, hence fetching messages records directly and live-update will be auto handled.